### PR TITLE
update zh translation /docs/reference/setup-tools/kubeadm/generated/k…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_token_delete.md [hunk 1]
```diff
@@ -85,56 +66,55 @@ kubeadm token delete [token-value]
...
-</table>
+<!--
+SEE ALSO
+-->
+查看其他
 
+<!--
+* [kubeadm token](kubeadm_token.md)	 - Manage bootstrap tokens
+-->
+* [kubeadm token](kubeadm_token.md)	 - 管理引导 token
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_token_delete.md]:
```suggestion
* [kubeadm token](kubeadm_token.md)	 - 管理引导令牌
```

**Comment by tanjunchen** [comment]
@xichengliudui  已经修改

**Pull request merged**